build/appveyor/build_mixxx.bat

```diff
@@ -1,5 +1,5 @@
-@echo on
-SETLOCAL
+@echo off
```

Comment:
Why did you change this?

Reply:
Because it is only needed if you are debugging at which line something has happened.
There's no reason to print all the commands that are being executed. In fact, reading the output might cause confusion sometimes, because it prints all the code of a conditional block, not only the exact lines executed.